Server/src/models/interfaces/CategoryInterface.ts

```diff
@@ -0,0 +1,4 @@
+export interface ICategory {

```

Comment:
If an interface is only related to models then we can keep it under /models/interfaces but if its generic then we have a folder for generic interfaces!

Reply:
After recent changes, these are exclusively related to models now